README.md

```diff
@@ -2,8 +2,45 @@
 
 RAPIDS - combined conda package for all of RAPIDS libraries
 
-## RAPIDS Meta-packages
+## RAPIDS Metapackages
 
-The conda recipe in the `conda` folder provides the RAPIDS meta-packages, which when installed will provide the latest RAPIDS libraries for the given version.
+The conda recipe in the `conda` folder provides the RAPIDS metapackages, which when installed will provide the latest RAPIDS libraries for the given version.
 
-See the [README](conda/recipes/README.md) for more information about the meta-packages and how to update versions.
+See the [README](conda/recipes/README.md) for more information about the metapackages and how to update versions.
+
+## dask Metapackage
+
+This repository provides metapackages for pip and conda that centralize the dask version dependency across RAPIDS.
+dask's API instability means that each RAPIDS release must pin to a very specific dask release to avoid incompatibilities.
```

Comment:
```suggestion
Dask's API instability means that each RAPIDS release must pin to a very specific dask release to avoid incompatibilities.
```

Reply:
Not relevant for the proposal itself, but this is something that truly confused me when I first found it out. The name `dask` (lower-case `d`) is _only_ used in the new logo, while `Dask` (capital `D`) should still be used in writing. See www.dask.org, it always uses `Dask` in writing. I think it's fair to use `dask` when specifically referring to a package (e.g., conda-forge or PyPI).